app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+
+  def create
+    @company = Company.find(params[:company_id])
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
```

Comment:
i suggest leveraging your associations a little more here - because a job has many comments, we could instead find the job first and rework this method to look like the following:

```ruby
def create
  @job = Job.find(params[:job_id])
  @comment = @job.comments.new(comment_params)
  if @comment.save
    redirect_to company_job_path(@job.company, @job)
  else
     redirect_to new_company_job_comment_path
  end
end
```